projects/plugins/jetpack/modules/subscriptions/views.php

```diff
@@ -52,23 +52,32 @@ function __construct() {
 			     is_customize_preview()
 		     )
 		) {
-			add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_style' ) );
+			add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
 		}
 	}
 
 	/**
-	 * Enqueue the form's CSS.
+	 * Enqueue the form's CSS and JS
 	 *
 	 * @since 4.5.0
 	 */
-	function enqueue_style() {
+	public function enqueue_scripts() {
```

Comment:
Since you're still enqueuing styles too, `assets` is a common term (`function enqueue_assets()`). Hm, did this method need to be made `public` to work? I'd actually think it _would_, but then were the styles not being enqueued before?..

Reply:
the linter wanted it to be public